.github/workflows/build-zip-file.yml

```diff
@@ -19,16 +44,26 @@ jobs:
         id: build_plugin
         uses: ./.github/actions/build
 
-      - name: "Unzip the file (prevents double zip issue)"
+      - name: "Add file size notice & delete build zip"
         run: |
-          echo ":information_source: Unzipping to a folder to prevent double-zip issue ([known limitation](https://github.com/actions/upload-artifact#zipped-artifact-downloads)). Increase in artifact size is expected since we're uploading the folder." >> $GITHUB_STEP_SUMMARY
-          unzip ${{ steps.build_plugin.outputs.release-filename }} -d zipfile
-      
+          echo ":information_source: Ignore the artifact size mentioned since GitHub calculates the size of the source folder instead of the zip file created." >> $GITHUB_STEP_SUMMARY
+          rm -rf release/${{ steps.build_plugin.outputs.release-filename }}
```

Comment:
Ah yes it's not needed. I added the deletion step because I ran the build script from inside the release folder, which moved the zip file inside the release folder rather than plugin root

I'll remove it.

Reply:
Removed in https://github.com/Automattic/woocommerce-payments/pull/5586/commits/b270fa50fc9a47d4041741b46c9374592d5e985b